src/Component.js

```diff
@@ -59,6 +67,19 @@ export class Component {
       this._pool.release(this);
     }
   }
+
+  checkUndefinedAttributes(src) {
+    const schema = this.constructor.schema;
+
+    // Check that the attributes defined in source are also defined in the schema
+    Object.keys(src).forEach(srcKey => {
+      if (!schema.hasOwnProperty(srcKey)) {
+        console.warn(
```

Comment:
Probably throwing an error? As you are trying to set an attribute and the attribute won't be set, so it seems like a important error